lib/src/component/fragment_component.dart

```diff
@@ -12,32 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'package:over_react/src/component_declaration/builder_helpers.dart'
-    as builder_helpers;
-import 'package:react/react_client.dart';
-import 'package:react/react_client/js_backed_map.dart';
-import 'package:react/react.dart' as react;
-
-class FragmentProps extends builder_helpers.UiProps {
-  // Initialize to a JsBackedMap so that copying can be optimized
-  // when converting props during ReactElement creation.
-  FragmentProps([Map props]) : this.props = props ?? JsBackedMap();
 
-  @override
-  ReactComponentFactoryProxy componentFactory = react.Fragment;
+import 'package:over_react/src/util/js_component.dart';
+import 'package:react/react.dart' as react;
+import 'package:react/react_client/js_backed_map.dart';
 
-  @override
-  final Map props;
+import '../component_declaration/builder_helpers.dart';
+import '../component_declaration/function_component.dart';
 
-  @override
-  String get propKeyNamespace => '';
+part 'fragment_component.over_react.g.dart';
 
-  @override
-  bool get $isClassGenerated => true;
-}
+@Props(keyNamespace: '')
+class FragmentProps extends UiProps {}
```

Comment:
Yeah, I changed it to match other JS components, since any props that would present wouldn't be namespaced.

It doesn't have any props though, so the key namespace doesn't really make a difference here